cmd/xk6/main.go

```diff
@@ -160,6 +160,15 @@ func runBuild(ctx context.Context, args []string) error {
 		}
 	}
 
+	fmt.Println("***************************************************")
+	fmt.Println("* BUILD COMPLETE - FOLLOW THESE STEPS TO CONTINUE *")
+	fmt.Println("***************************************************")
+	fmt.Println("xk6 has now produced a new k6 binary with the following:")
+	for _, extension := range extensions {
+		fmt.Println(" -", extension.PackagePath)
+	}
+	fmt.Printf("To use it, call it directly as `%v run <SCRIPT_NAME>`.\n", output)
```

Comment:
Hhmm I think we'll need to do something more clever than dumping `output` here.

Consider:

```sh
$ ./xk6 build --output xk6-browser --with github.com/grafana/xk6-browser
[...]
xk6 has now produced a new k6 binary with the following:
 - github.com/grafana/xk6-browser
To use it, call it directly as `xk6-browser run <SCRIPT_NAME>`.
$ xk6-browser
zsh: command not found: xk6-browser
```

Maybe we need to check if it's in the CWD and prefix `./`? But then that would be wrong for Windows... :-/

I know this is useful for new users, but ensuring it works in all scenarios can get messy. :disappointed:

Reply:
> ensuring it works in all scenarios can get messy. 😞

Indeed. 

Not 100% sure if the output file, if overridden with `--output`, will even include the `.exe` on Windows if not directly provided 🤷

Reply:
I just tested it, and it doesn't. The file is written exactly as specified, so if the user doesn't include the `.exe` extension, it will be unusable :facepalm:

We might consider fixing this, but TBH whoever's using `--output` will be able to figure it out and just include the extension the second time or rename the binary. I'd rather us not try to be smart about it.